.buildkite/scripts/publish_docker_studio.ps1

```diff
@@ -21,6 +21,9 @@ Write-Host "--- Publishing the Windows Docker Studio"
 . ./results/last_image.ps1
```

Comment:
I can't comment directly, but what if line 18 fails?

Reply:
we have a check on the docker build but although the `docker tag`s are pretty safe, we might as well guard them.